pkg/services/blockfetcher/blockfetcher.go

```diff
@@ -204,7 +218,9 @@ func (bfs *Service) oidDownloader() {
 	}
 	var force bool
 	if err != nil {
-		bfs.log.Error("NeoFS BlockFetcher service: OID downloading routine failed", zap.Error(err))
+		if bfs.IsActive() {
+			bfs.log.Error("NeoFS BlockFetcher service: OID downloading routine failed", zap.Error(err))
+		}
```

Comment:
if you interrupt the blockfetcher it will return ERRORs in the log from retries before full shutdown:
```
2024-11-26T17:12:31.522+0300	INFO	persisted to disk	{"blocks": 546, "keys": 10498, "headerHeight": 4781928, "blockHeight": 4781928, "took": "28.545042ms"}
2024-11-26T17:12:31.837+0300	INFO	shutting down server	{"peers": 14}
2024-11-26T17:12:31.837+0300	INFO	shutting down NeoFS BlockFetcher service	{"force": true}
2024-11-26T17:12:31.837+0300	ERROR	failed to objectGet block	{"oid": "GFu9KXHvJppnViDoGcq47w3mvtMjgPcvufhcFvu2zWJY", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.837+0300	ERROR	failed to objectGet block	{"oid": "7YVVB5s5J8yeCi9CiEESQjVuTc8vA4mGTBTQZzsNmaSH", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.837+0300	ERROR	failed to objectGet block	{"oid": "6nuEiHEhKCW9cotSgUSEPrQhBC66C4FNsGQqMkPZhGtq", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.837+0300	ERROR	failed to objectGet block	{"oid": "7D3yhgXCEp5ws5ek1wM2BjmrsngcodqvNzzwmZt88svQ", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.837+0300	ERROR	failed to objectGet block	{"oid": "GB8tB5NHkwt66dPVsPqpSmbGh3Mvf6KAUd2pPhXi8PDL", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.837+0300	ERROR	failed to objectGet block	{"oid": "5kWCLgqPLStjNrExk2ByeTCXzg4rR6ogS1dNyWZiA4Ed", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.837+0300	ERROR	failed to objectGet block	{"oid": "Dryg8epnQh2nb7EjGwS9EJwSWe5X8jUZ5DecvnxuKaZi", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.837+0300	ERROR	failed to objectGet block	{"oid": "9kfzfSspnLmtrnEjybSgEqAPxRE2hxN5W5UwPWsFeYn4", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.837+0300	ERROR	failed to objectGet block	{"oid": "6P5LCKn1tvXc3idjAouAvULRnp9CuD3WPp1ygYZEuXiZ", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.837+0300	ERROR	failed to objectGet block	{"oid": "643yyN2Y3TRSDHFTm1LRX7d7N334zZMNx18qYd9zbsvE", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.837+0300	ERROR	failed to objectGet block	{"oid": "GYr26RrRxhbvfqZUyJwi6rabSpwvaXQisTLbucgxUReX", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.837+0300	ERROR	failed to objectGet block	{"oid": "E7tkKzEwcUizGXJjJ6U4tQt5MyWUVMk6KSRthmLJbEDs", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.837+0300	ERROR	failed to objectGet block	{"oid": "Har2cnFurMMYw4hJdS4f6CbnTaxKct94GQGEoXJae333", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.837+0300	ERROR	failed to objectGet block	{"oid": "92v1PQoxwY1kVUYiUK5ZvfRJ5SPrkDq1SFaJjjqEvend", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.837+0300	ERROR	failed to objectGet block	{"oid": "CvTnFcP1QZYChDG9tzQkk1yK9CvjoDDwahtAp1P6jpCV", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.837+0300	ERROR	failed to objectGet block	{"oid": "63uTvmUXfwdpkNDQRprw8ojmyteUncX6SMFuc7zEtaWq", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.837+0300	ERROR	failed to objectGet block	{"oid": "84H6NuUh7Shcyrte5eH1QWYTtnHhJf2YxHNaXQbZcwqb", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.837+0300	ERROR	failed to objectGet block	{"oid": "9WF49WRraHtmjorXY3LwzmFGb9ecUtmLPrBUeqceQRKN", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.837+0300	ERROR	failed to objectGet block	{"oid": "BKYP7J1p1y1CXL2SCKifCstFSMgNATbeMyUegVHdXY2M", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.837+0300	ERROR	failed to objectGet block	{"oid": "2BwazDAVHtcpZLZ2kKQTXfmq7CFL7MfTcLsSsToVXHjG", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.837+0300	ERROR	failed to objectGet block	{"oid": "D6WMM3dr3TbVP7SYWswmuzNcBGmVVw6vCgXsp7Wx6FoK", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.837+0300	ERROR	failed to objectGet block	{"oid": "69gh49bm7D7yJ8VGRSteH2Ameqy9QYGNJzsAEzG7UGQD", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "EwCKLUKjuRPX9nJ2yuLD33tZPTJudBZ3X3Kp4bK9Jx6H", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.837+0300	ERROR	failed to objectGet block	{"oid": "AHxogVNg9BZD7M2eNosx5GAg8pG7xdz2C6LA2vrrKw1C", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.837+0300	ERROR	failed to objectGet block	{"oid": "CuP5uQMavLHQTRxWWawk1va2Wn2C6ZuC4e1CVMey6CPa", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.837+0300	ERROR	failed to objectGet block	{"oid": "3WP4LJDmZ1bT1LG77P5Dj334D34btVUwTo2nzN9xfpFt", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.837+0300	ERROR	failed to objectGet block	{"oid": "C4P63EhVH2rCGyBsFydSG2RAXxgxF8SFLfqXXDuraaUB", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "8LipZNPTX3Ur3CWW4AZrV6b8m111HrZdNLsx1yqyoX6y", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "HwhjvPEFCAziE1FjdVfqsyqBfogz2Sw2TEv5RPGpU55i", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "AjjBrrvwgVUnJB3UVmU7Xd296f9T7MYG46y9eeKTnDav", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.837+0300	ERROR	failed to objectGet block	{"oid": "6BF9hVSrz1ptaEBK5e7hKHt5rERkDMUstBjm9R8bHnoP", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "CzF6VceVBYdAKdG9wGERrPFB9PZybdnxWqxudqhZXTQj", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "H2yvmZU3MQ6Jo8KWEQm6CmmJa3uAuRhS4c1cvJMgLBtL", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "F7KiQnPXv6v6dsDsE3cKXwHNkgju2teLrcbX5deqKNgr", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "4GcHmht9mijrW6fe1jGfzKaznTrJRJk6EwisDd6WLRsm", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "G8sdcXRJZSCcWie77PHNsf7SzRQZR3LGAdt3WC6zkdL8", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "4N6umV3w1frtf85eLo9q7ve9oE81dcA512mXWYjsr4rA", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "pWALRGWuidN3SfaUC6zcVTq6M9KAgDt1B47RuZAReo3", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "6JiqJJAHX6ptgF9NKLvUCsgYGLSYXVHcWGVhU1gbdx9L", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "7V1QF7ByejrrpR5x3MDRt62CQQ6zXeVN8hGViFsmYfDF", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "AZK4kZxYNn1SmeEkk6j32cpazMPnbtnmTUH4a2PUn9KV", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "aWgL4gL5GBeKe6v8gRjzAQKFrJ9snNZioTTbVvRyTht", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "Fbnt3Ztw2iwpemrFEvTWN9ZiKf6n7tdorgt7oeuRWLfj", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "4USP54HrEbxcDtPYbaASzcRxoC95UBoYAbJPtzd7Qiri", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "3y67ERs6vYBE97WAtXuzXC9qowSgW1GxAp8Aju1SiVQg", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "4fVbw42KnDvq3B8kMwXJoeWu26RhwDryF3pogP6vjCKJ", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "HzgVJyMYLzpvSgQUNbq9fnam2uMjyvG2neHZeMSwd4SR", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "D3hRgtUJSDYgE9Ycze5xQbE2cBnZ6tbmd4555RARiYse", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "6Arw1Kbdu82mLyMAtSYeua1L8jdYtNrHF9smaC4kusTF", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "G1kwoJUgvmNAXxiQQoG7Tt8hdaY3hiQFv2EwuJDu9bbn", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "EJ8Wh18Wiky9evTfueSVb8y2HFW3g2vTnWD97s8JTQkV", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "EiMxyYt1EsQJXJ6oVBPESsbuSGkYibQ3eXANWZ13dcHg", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "711ZPxRgGp2yyHtgkLrBuQaLV1pxByBWo3z4vkQHewtS", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "HoYvikLGo8GgFudniaGhFE97YubetuiPRzLK8pGJ7Gyn", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "5M7nx1z9btHQ4SiAPMoeEKgGcvFe5bLcaoSLYQJR4Cds", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "7DF9Av3fQUkiKR3m4EKZo5PPwdgS9a7x7nL7vb9yjUmV", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "Bxh7pE1By2Rzkq11A8UBDyQkrwXP3GWSdY8BDDEzkLBz", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "QK3nApFpBs6HHNjfkjCSMzVJCRVFsWEBzpr2Tm3iA3j", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "Ds6jownm9BaKebS3mQh98EQA8zgUbPs9cw9VfmhM5mBa", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "BgugysdoP922TeFTpwrhYdHyfA8g8U8NA6KGUcJkj5At", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "73k4h91MCVJDAntJLSJPagjvTSLZGMdfefPSvDXFHogM", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "F6dbkzK6hTtcXabuLyVr8WPmAdaxygxk7JxbTK41L1AN", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "8ezQkuXQnQywQ8xTbPF8UguSondwmww6ZL63zyQZhr8A", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "2ouTxHGw1sH6pLQGkYwE64quZ3X55nShC5TM3sviWzSH", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "FVBkGKMRQTVnL4JExLoL7wUFso6KqRVmLLpFk6BtQ7iA", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "GCpAzzG4Uri6UmTkBDAb3cR8TjzL6ra8k9NhR21wykRr", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "BtVt3ut1jByX75tHKPqgxjHWUJBEaswz9yDLcrdPvaFB", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "9HcLJ6Qm6DjPvyTKgiLRdqAu3agpasZbxw5oV8nHBTBL", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "5GPRAAEnj9EGakTfMGXBFckVQE1GUUWR7RPfTrwqbVSZ", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "8mGQenC1brYdJjWp9J9RBNc5ph74LBaNGAASJ8Tuh1Wt", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "Cwx3TJ1hx6AxTxnjKnc7Tu3wqcFTmi9MRkNGd2QYCTr7", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "HTSDR8Z67sFgi7vBKttp3YeaA87j87eZq8R3hY57ixof", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "5ETtp5LwSqFgAZ1y8cor6M9LLb87Ur6XtbmG53XQJHD7", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.839+0300	ERROR	failed to objectGet block	{"oid": "9aoEPKsdVUEGeqgKJN37YwAD3fdFZRXcgiRnAn7Gr9Fv", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.839+0300	ERROR	failed to objectGet block	{"oid": "9SBz5jVeV4MoEdMRzG3kh5SfceMBgRJUiSNQmNmy2bds", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "3oUJk8nBPSy7yT6Ba5wGJwRBfT7LGDcWE5Nmk51qSMe3", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.838+0300	ERROR	failed to objectGet block	{"oid": "4gKHkDaffsEjfPwQP1Q7ZThoukpbEaZcujcX8boKRHCG", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.839+0300	ERROR	failed to objectGet block	{"oid": "DjbP3PPrtkRU8yHetGytZhhPGdAkXtt3tHhNJXeQBjJ7", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.839+0300	ERROR	failed to objectGet block	{"oid": "J9q6QuyRfmCrNcL1SjqENKkqZLJCJmWBXsEbX8hJoui7", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.839+0300	ERROR	failed to objectGet block	{"oid": "6DzsrUBokbC3m17TBW1xkoTTRA7kBzbtTkUGPrCmboro", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.839+0300	ERROR	failed to objectGet block	{"oid": "Gaejv1hamV9DY7Yr7QBqCgoisALASqLFjQL81KTiS7Vq", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.839+0300	ERROR	failed to objectGet block	{"oid": "5RYT5Ftf77HSjDJtuHjJstXHLPrQ4uWiG3WEykLmjhVn", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.839+0300	ERROR	failed to objectGet block	{"oid": "22rgDqZT74Gq73Xsr9Vu96KGmR4egcyM1fvz7mrvAWBA", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.839+0300	ERROR	failed to objectGet block	{"oid": "8XsWWoxP72tjattP1SgEbWfuy2NTH3NXmbV7b8FfBFf8", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.839+0300	ERROR	failed to objectGet block	{"oid": "7qqgUXyQmCwnerKr6H5SwrteGSRmfvpNJmLwxzPqnHv2", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.839+0300	ERROR	failed to objectGet block	{"oid": "55aHnWqsKV5g7tABEGhFDNF41J1wNuk3K9F23CM4Piyx", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.839+0300	ERROR	failed to objectGet block	{"oid": "4UkkUtTYsyLZGCfivM7mfpwacNM13jDnZM7YAUmBG584", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.839+0300	ERROR	failed to objectGet block	{"oid": "GesNBJoKU8fdD7Mat6hHqesAdKtRFGTiBUdZy6uPUrrq", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.839+0300	ERROR	failed to objectGet block	{"oid": "2ewFfDpEfAhbyqNA9zqDoeKHHV8ufC9sKYL5H7Gwk3EC", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.839+0300	ERROR	failed to objectGet block	{"oid": "9ZL6vhNqYjhjuKYggNsWdkx5p3aoyHMB3qowVUtrXxv6", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.839+0300	ERROR	failed to objectGet block	{"oid": "Du9AnB657v21UB54t4KUBtzhra5f5KBdpJEvWzkEsqCQ", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.839+0300	ERROR	failed to objectGet block	{"oid": "B462ERuuqvcfMsPXgWQ8CJ4wEVD9UyWfZq1An92x5pHV", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.839+0300	ERROR	failed to objectGet block	{"oid": "33RfQxPkE8TkYNf2MyMDPqiMANTi5yQXjC6Xj5BMS5JS", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.839+0300	ERROR	failed to objectGet block	{"oid": "D9NAmb7A9WrV6RLdCwE8fe1N2uruLgc3aYnXoNuHDmTM", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.839+0300	ERROR	failed to objectGet block	{"oid": "EBFZA6h9xCLa2XqgYppVkXZYkLUcfD2T8d4J1rPoDw8D", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.839+0300	ERROR	failed to objectGet block	{"oid": "6qFuWDFF8ysPCKzTaAfbpjeLEwQsQq5vbDErHsSCWUHy", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.839+0300	ERROR	failed to objectGet block	{"oid": "M1R7usi5niCDNGZo1tL37QL3jaTQvKfCuvRwBNgGzXh", "error": "service became inactive during backoff, aborting retry"}
2024-11-26T17:12:31.840+0300	ERROR	failed to objectGet block	{"oid": "85BnhfU7vaV3A7zt3yp2MH7KMwJiaDnxNSXJKu5D8HRM", "error": "service is not active, aborting retry"}
2024-11-26T17:12:31.845+0300	INFO	peer disconnected	{"addr": "139.59.211.37:20333", "error": "server shutdown", "peerCount": 13}
2024-11-26T17:12:31.845+0300	INFO	peer disconnected	{"addr": "34.126.65.191:20333", "error": "server shutdown", "peerCount": 12}
2024-11-26T17:12:31.845+0300	INFO	peer disconnected	{"addr": "35.238.26.128:20333", "error": "server shutdown", "peerCount": 11}
2024-11-26T17:12:31.845+0300	INFO	peer disconnected	{"addr": "34.133.235.69:20333", "error": "server shutdown", "peerCount": 10}
2024-11-26T17:12:31.845+0300	INFO	peer disconnected	{"addr": "35.240.246.253:20333", "error": "server shutdown", "peerCount": 9}
2024-11-26T17:12:31.846+0300	INFO	peer disconnected	{"addr": "35.232.23.196:20333", "error": "server shutdown", "peerCount": 8}
2024-11-26T17:12:31.846+0300	INFO	peer disconnected	{"addr": "157.90.177.38:30333", "error": "server shutdown", "peerCount": 7}
2024-11-26T17:12:31.846+0300	INFO	peer disconnected	{"addr": "49.12.226.190:21333", "error": "server shutdown", "peerCount": 6}
2024-11-26T17:12:31.846+0300	INFO	peer disconnected	{"addr": "35.192.59.217:20333", "error": "server shutdown", "peerCount": 5}
2024-11-26T17:12:31.846+0300	INFO	peer disconnected	{"addr": "35.188.199.101:20333", "error": "server shutdown", "peerCount": 4}
2024-11-26T17:12:31.846+0300	INFO	peer disconnected	{"addr": "34.87.58.201:20333", "error": "server shutdown", "peerCount": 3}
2024-11-26T17:12:31.846+0300	INFO	peer disconnected	{"addr": "35.226.51.66:20333", "error": "server shutdown", "peerCount": 2}
2024-11-26T17:12:31.846+0300	INFO	peer disconnected	{"addr": "5.78.81.79:21333", "error": "server shutdown", "peerCount": 1}
2024-11-26T17:12:31.846+0300	INFO	peer disconnected	{"addr": "34.124.145.177:20333", "error": "server shutdown", "peerCount": 0}
2024-11-26T17:12:31.846+0300	INFO	shutting down RPC server	{"endpoint": "[::]:20332"}
2024-11-26T17:12:31.853+0300	INFO	shutting down service	{"service": "Prometheus", "endpoint": "[::]:2112"}
```

so to prevent such errors if bfs.IsActive()  is used.